### PR TITLE
Move logging-related args under logging args

### LINKED ISSUE #115: Move logging-related args under logging args
In [config_handler.py](https://github.com/uptrain-ai/uptrain/blob/main/uptrain/core/classes/helpers/config_handler.py), remove logging-related arguments like `st_logging` and `log_data` to the class `LoggingArgs`.

This is a good first issue to get started with UpTrain.

### PR DESCRIPTION
**Pull request by scorcism**
fixes #115

### CODE REVIEW

**Comment by scorcism** [comment]
Now changing
`"st_logging": True,` **to** `"logging_args": {"st_logging": True},`

**Comment by scorcism** [comment]
In **test_dashbard** I have merged both as you can see here: https://imgur.com/uWMGoWM.png is it okay ?

**Comment by vipgupta** [comment]
Yes, that's good!

**Comment by scorcism** [comment]
okay, can you let me know about reame one.

**Comment by vipgupta** [comment]
2nd point looks good. I will make the rest of the changes.

**Comment by scorcism** [comment]
okay i'm pushing the final one

**Comment by scorcism** [comment]
DONE

File: uptrain/core/classes/framework.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def __init__(self, cfg_dict={}):
             shutil.rmtree(self.fold_name)
         os.mkdir(self.fold_name)
 
-        self.log_data = cfg.log_data
```
Comment (vipgupta) [inline uptrain/core/classes/framework.py:65]: Typo here. Please fix and update.

Comment (vipgupta) [reply uptrain/core/classes/framework.py:65]: I fixed it

**Comment by scorcism** [comment]
Anythig left to be done in this PR ?

**Review by vipgupta — APPROVED** [review]

**Pull request merged**